support-frontend/assets/helpers/abTests/abtest.js

```diff
@@ -344,10 +352,14 @@ const init = (
   const participations: Participations = getParticipations(abTests, mvt, country, countryGroupId);
   const urlParticipations: ?Participations = getParticipationsFromUrl();
   const serverSideParticipations: ?Participations = getServerSideParticipations();
+  const amountsTestParticipations: ?Participations = getAmountsTestParticipations(countryGroupId, settings);
+  const localStorageParticipations: Participations = getLocalStorageParticipations();
   const combinedParticipations: Participations = {
     ...participations,
     ...urlParticipations,
     ...serverSideParticipations,
+    ...amountsTestParticipations,
+    ...localStorageParticipations,
```

Comment:
@tomrf1 I've tried this instead. Before, the `localStorageParticipations` were only overwritting the participations coming from `abTestDefinitions.js`. Now by putting it here, you can overwrite any participations

Reply:
although maybe url participations should take come last? (highest priority)

Reply:
Yes you'd want the url to override. How's this?
```
    ...participations,
    ...amountsTestParticipations,
    ...serverSideParticipations,
    ...localStorageParticipations,
    ...urlParticipations,
```

Reply:
just pushed that reorder up!

Reply:
thanks!